REPORT.md

```diff
@@ -341,16 +660,723 @@ vainfo: Supported profile and entrypoints
+DIVX5_MOBILE                    0  8192  2048  2048
+DIVX5_HOME_THEATER              0  8192  2048  2048
+DIVX5_HD_1080P                  0  8192  2048  2048
+H264_CONSTRAINED_BASELINE      51 65536  4096  4096
+H264_EXTENDED                  51 65536  4096  4096
+H264_PROGRESSIVE_HIGH          51 65536  4096  4096
+H264_CONSTRAINED_HIGH          51 65536  4096  4096
+H264_HIGH_444_PREDICTIVE       51 65536  4096  4096
+VP9_PROFILE_0                   1 262144  8192  8192
+VP9_PROFILE_1                  --- not supported ---
+VP9_PROFILE_2                   1 262144  8192  8192
+VP9_PROFILE_3                  --- not supported ---
+HEVC_MAIN                      153 262144  8192  8192
+HEVC_MAIN_10                   153 262144  8192  8192
+HEVC_MAIN_STILL                --- not supported ---
+HEVC_MAIN_12                   153 262144  8192  8192
+HEVC_MAIN_444                  153 262144  8192  8192
+HEVC_MAIN_444_10               153 262144  8192  8192
+HEVC_MAIN_444_12               153 262144  8192  8192
+
+Output surface:
+
+name              width height nat types
+----------------------------------------------------
+B8G8R8A8         32768 32768    y  Y8U8V8A8 V8U8Y8A8 A4I4 I4A4 A8I8 I8A8 
+R10G10B10A2      32768 32768    y  Y8U8V8A8 V8U8Y8A8 A4I4 I4A4 A8I8 I8A8 
+
+Bitmap surface:
+
+name              width height
+------------------------------
+B8G8R8A8         32768 32768
+R8G8B8A8         32768 32768
+R10G10B10A2      32768 32768
+B10G10R10A2      32768 32768
+A8               32768 32768
+
+Video mixer:
+
+feature name                    sup
+------------------------------------
+DEINTERLACE_TEMPORAL             y
+DEINTERLACE_TEMPORAL_SPATIAL     y
+INVERSE_TELECINE                 y
+NOISE_REDUCTION                  y
+SHARPNESS                        y
+LUMA_KEY                         y
+HIGH QUALITY SCALING - L1        y
+HIGH QUALITY SCALING - L2        -
+HIGH QUALITY SCALING - L3        -
+HIGH QUALITY SCALING - L4        -
+HIGH QUALITY SCALING - L5        -
+HIGH QUALITY SCALING - L6        -
+HIGH QUALITY SCALING - L7        -
+HIGH QUALITY SCALING - L8        -
+HIGH QUALITY SCALING - L9        -
+
+parameter name                  sup      min      max
+-----------------------------------------------------
+VIDEO_SURFACE_WIDTH              y         1     8192
+VIDEO_SURFACE_HEIGHT             y         1     8192
+CHROMA_TYPE                      y  
+LAYERS                           y         0        4
+
+attribute name                  sup      min      max
+-----------------------------------------------------
+BACKGROUND_COLOR                 y  
+CSC_MATRIX                       y  
+NOISE_REDUCTION_LEVEL            y      0.00     1.00
+SHARPNESS_LEVEL                  y     -1.00     1.00
+LUMA_KEY_MIN_LUMA                y  
+LUMA_KEY_MAX_LUMA                y  
+```
+
+### H.266/VVC Nvidia
+
+`./fluster.py run -d VVdeC-H266 GStreamer-H.266-VVdec-Gst1.0 FFmpeg-H.266 -ts JVET-VVC_draft6 -s`
```

Comment:
Same as for Intel

> In the list below i see no results for `VVdeC-H266` and `GStreamer-H.266-VVdec-Gst1.0`.
> - for the first one: did you build the executable of the reference decoder using the Makefile (target: `h266_reference_decoder`)?
> - for the second one: your GStreamer version might not be new enough to support the decoder.
> 
> If this is the case can you
> - build the ref decoder and add the test results below?
> - simply remove the decoder from this line, otherwise add the test results below?